### ISSUE TITLE
Throwing `NotFoundHttpException` in form constructor causes recursion with path-based breadcrumbs on 404/403 pages

### ISSUE THREAD

**Issue by Hydraner** [systopia/civiremote#9]
Edit by @jensschuppe:

> There is a recursion when the constructor of the `RegisterForm` throws a `NotFoundHttpException` and the resulting 404 page includes a breadcrumb; the `PathBasedBreadcrumbBuilder` tries to resolve the title of the event registration form and causes the constructor again to issue a `NotFoundHttpException` - that's where the recursion starts.

Here tThe stacktrace. The custom PathBasedBreadcrumbBuilder is not doing anything here but calling the parent PathBasedBreadcrumbBuilder from core.

The result is a recursion.
```
RegisterForm.php:122, Drupal\civiremote_event\Form\RegisterForm->__construct()
RegisterForm.php:180, Drupal\civiremote_event\Form\RegisterForm::create()
ClassResolver.php:28, Drupal\Core\DependencyInjection\ClassResolver->getInstanceFromDefinition()
ControllerResolver.php:116, Drupal\Core\Controller\ControllerResolver->createController()
ControllerResolver.php:68, Drupal\Core\Controller\ControllerResolver->getControllerFromDefinition()
CustomAccessCheck.php:65, Drupal\Core\Access\CustomAccessCheck->access()
AccessManager.php:160, call_user_func_array:{/var/www/html/web/core/lib/Drupal/Core/Access/AccessManager.php:160}()
AccessManager.php:160, Drupal\Core\Access\AccessManager->performCheck()
AccessManager.php:136, Drupal\Core\Access\AccessManager->check()
AccessManager.php:113, Drupal\Core\Access\AccessManager->checkRequest()
AccessAwareRouter.php:104, Drupal\Core\Routing\AccessAwareRouter->checkAccess()
AccessAwareRouter.php:89, Drupal\Core\Routing\AccessAwareRouter->matchRequest()
PathBasedBreadcrumbBuilder.php:225, Drupal\system\PathBasedBreadcrumbBuilder->getRequestForPath()
PathBasedBreadcrumbBuilder.php:171, Drupal\system\PathBasedBreadcrumbBuilder->build()
PathBasedBreadcrumbBuilder.php:21, Drupal\bwp_base\Breadcrumb\PathBasedBreadcrumbBuilder->build()
BreadcrumbManager.php:83, Drupal\Core\Breadcrumb\BreadcrumbManager->build()
…
```

**Comment by Hydraner** [comment]
```
CustomAccessCheck.php:65, Drupal\Core\Access\CustomAccessCheck->access()
AccessManager.php:160, call_user_func_array:{/var/www/html/web/core/lib/Drupal/Core/Access/AccessManager.php:160}()
AccessManager.php:160, Drupal\Core\Access\AccessManager->performCheck()
AccessManager.php:136, Drupal\Core\Access\AccessManager->check()
AccessManager.php:113, Drupal\Core\Access\AccessManager->checkRequest()
AccessAwareRouter.php:104, Drupal\Core\Routing\AccessAwareRouter->checkAccess()
AccessAwareRouter.php:89, Drupal\Core\Routing\AccessAwareRouter->matchRequest()
PathBasedBreadcrumbBuilder.php:225, Drupal\system\PathBasedBreadcrumbBuilder->getRequestForPath()
PathBasedBreadcrumbBuilder.php:171, Drupal\system\PathBasedBreadcrumbBuilder->build()
PathBasedBreadcrumbBuilder.php:21, Drupal\bwp_base\Breadcrumb\PathBasedBreadcrumbBuilder->build()
BreadcrumbManager.php:83, Drupal\Core\Breadcrumb\BreadcrumbManager->build()
SystemBreadcrumbBlock.php:72, Drupal\system\Plugin\Block\SystemBreadcrumbBlock->build()
BlockPluginViewBuilder.php:218, Drupal\block_plugin_view_builder\BlockPluginViewBuilder::preRender()
DoTrustedCallbackTrait.php:101, call_user_func_array:{/var/www/html/web/core/lib/Drupal/Core/Security/DoTrustedCallbackTrait.php:101}()
DoTrustedCallbackTrait.php:101, Drupal\Core\Render\Renderer->doTrustedCallback()
Renderer.php:788, Drupal\Core\Render\Renderer->doCallback()
Renderer.php:374, Drupal\Core\Render\Renderer->doRender()
Renderer.php:446, Drupal\Core\Render\Renderer->doRender()
Renderer.php:204, Drupal\Core\Render\Renderer->render()
TwigExtension.php:477, Drupal\Core\Template\TwigExtension->escapeFilter()
15:82, __TwigTemplate_d8721f0f6aebbd5bb04850254f3787b8->block_block_content()
Template.php:171, Twig\Template->displayBlock()
15:61, __TwigTemplate_35b0ad2b37afc89a549c145babd54d0a->doDisplay()
Template.php:394, Twig\Template->displayWithErrorHandling()
Template.php:367, Twig\Template->display()
15:45, __TwigTemplate_d8721f0f6aebbd5bb04850254f3787b8->doDisplay()
Template.php:394, Twig\Template->displayWithErrorHandling()
Template.php:367, Twig\Template->display()
Template.php:379, Twig\Template->render()
TemplateWrapper.php:40, Twig\TemplateWrapper->render()
twig.engine:53, twig_render_template()
ThemeManager.php:372, Drupal\Core\Theme\ThemeManager->render()
Renderer.php:433, Drupal\Core\Render\Renderer->doRender()
Renderer.php:446, Drupal\Core\Render\Renderer->doRender()
Renderer.php:204, Drupal\Core\Render\Renderer->render()
TwigExtension.php:477, Drupal\Core\Template\TwigExtension->escapeFilter()
15:92, __TwigTemplate_0393aa411e8c560ed6e7ffe204e1d3df->doDisplay()
Template.php:394, Twig\Template->displayWithErrorHandling()
Template.php:367, Twig\Template->display()
Template.php:379, Twig\Template->render()
TemplateWrapper.php:40, Twig\TemplateWrapper->render()
twig.engine:53, twig_render_template()
ThemeManager.php:372, Drupal\Core\Theme\ThemeManager->render()
Renderer.php:433, Drupal\Core\Render\Renderer->doRender()
Renderer.php:204, Drupal\Core\Render\Renderer->render()
TwigExtension.php:477, Drupal\Core\Template\TwigExtension->escapeFilter()
15:86, __TwigTemplate_6a92af3dbed8ce6ee06c2517506d319e->doDisplay()
Template.php:394, Twig\Template->displayWithErrorHandling()
Template.php:367, Twig\Template->display()
Template.php:379, Twig\Template->render()
TemplateWrapper.php:40, Twig\TemplateWrapper->render()
twig.engine:53, twig_render_template()
ThemeManager.php:372, Drupal\Core\Theme\ThemeManager->render()
Renderer.php:433, Drupal\Core\Render\Renderer->doRender()
Renderer.php:204, Drupal\Core\Render\Renderer->render()
HtmlRenderer.php:158, Drupal\Core\Render\MainContent\HtmlRenderer->Drupal\Core\Render\MainContent\{closure:/var/www/html/web/core/lib/Drupal/Core/Render/MainContent/HtmlRenderer.php:153-159}()
Renderer.php:580, Drupal\Core\Render\Renderer->executeInRenderContext()
HtmlRenderer.php:159, Drupal\Core\Render\MainContent\HtmlRenderer->renderResponse()
MainContentViewSubscriber.php:90, Drupal\Core\EventSubscriber\MainContentViewSubscriber->onViewRenderArray()
ContainerAwareEventDispatcher.php:111, call_user_func:{/var/www/html/web/core/lib/Drupal/Component/EventDispatcher/ContainerAwareEventDispatcher.php:111}()
ContainerAwareEventDispatcher.php:111, Drupal\Component\EventDispatcher\ContainerAwareEventDispatcher->dispatch()
HttpKernel.php:168, Symfony\Component\HttpKernel\HttpKernel->handleRaw()
HttpKernel.php:74, Symfony\Component\HttpKernel\HttpKernel->handle()
Session.php:58, Drupal\Core\StackMiddleware\Session->handle()
KernelPreHandle.php:48, Drupal\Core\StackMiddleware\KernelPreHandle->handle()
PageCache.php:106, Drupal\page_cache\StackMiddleware\PageCache->pass()
PageCache.php:85, Drupal\page_cache\StackMiddleware\PageCache->handle()
Cors.php:53, Asm89\Stack\Cors->handle()
ReverseProxyMiddleware.php:48, Drupal\Core\StackMiddleware\ReverseProxyMiddleware->handle()
NegotiationMiddleware.php:51, Drupal\Core\StackMiddleware\NegotiationMiddleware->handle()
StackedHttpKernel.php:51, Drupal\Core\StackMiddleware\StackedHttpKernel->handle()
DefaultExceptionHtmlSubscriber.php:166, Drupal\Core\EventSubscriber\DefaultExceptionHtmlSubscriber->makeSubrequest()
DefaultExceptionHtmlSubscriber.php:122, Drupal\Core\EventSubscriber\DefaultExceptionHtmlSubscriber->on404()
HttpExceptionSubscriberBase.php:109, Drupal\Core\EventSubscriber\HttpExceptionSubscriberBase->onException()
ContainerAwareEventDispatcher.php:111, call_user_func:{/var/www/html/web/core/lib/Drupal/Component/EventDispatcher/ContainerAwareEventDispatcher.php:111}()
ContainerAwareEventDispatcher.php:111, Drupal\Component\EventDispatcher\ContainerAwareEventDispatcher->dispatch()
HttpKernel.php:221, Symfony\Component\HttpKernel\HttpKernel->handleThrowable()
HttpKernel.php:89, Symfony\Component\HttpKernel\HttpKernel->handle()
Session.php:58, Drupal\Core\StackMiddleware\Session->handle()
KernelPreHandle.php:48, Drupal\Core\StackMiddleware\KernelPreHandle->handle()
PageCache.php:106, Drupal\page_cache\StackMiddleware\PageCache->pass()
PageCache.php:85, Drupal\page_cache\StackMiddleware\PageCache->handle()
Cors.php:53, Asm89\Stack\Cors->handle()
ReverseProxyMiddleware.php:48, Drupal\Core\StackMiddleware\ReverseProxyMiddleware->handle()
NegotiationMiddleware.php:51, Drupal\Core\StackMiddleware\NegotiationMiddleware->handle()
StackedHttpKernel.php:51, Drupal\Core\StackMiddleware\StackedHttpKernel->handle()
DefaultExceptionHtmlSubscriber.php:166, Drupal\Core\EventSubscriber\DefaultExceptionHtmlSubscriber->makeSubrequest()
DefaultExceptionHtmlSubscriber.php:122, Drupal\Core\EventSubscriber\DefaultExceptionHtmlSubscriber->on404()
HttpExceptionSubscriberBase.php:109, Drupal\Core\EventSubscriber\HttpExceptionSubscriberBase->onException()
ContainerAwareEventDispatcher.php:111, call_user_func:{/var/www/html/web/core/lib/Drupal/Component/EventDispatcher/ContainerAwareEventDispatcher.php:111}()
ContainerAwareEventDispatcher.php:111, Drupal\Component\EventDispatcher\ContainerAwareEventDispatcher->dispatch()
HttpKernel.php:221, Symfony\Component\HttpKernel\HttpKernel->handleThrowable()
HttpKernel.php:89, Symfony\Component\HttpKernel\HttpKernel->handle()
Session.php:58, Drupal\Core\StackMiddleware\Session->handle()
KernelPreHandle.php:48, Drupal\Core\StackMiddleware\KernelPreHandle->handle()
PageCache.php:106, Drupal\page_cache\StackMiddleware\PageCache->pass()
PageCache.php:85, Drupal\page_cache\StackMiddleware\PageCache->handle()
Cors.php:53, Asm89\Stack\Cors->handle()
ReverseProxyMiddleware.php:48, Drupal\Core\StackMiddleware\ReverseProxyMiddleware->handle()
NegotiationMiddleware.php:51, Drupal\Core\StackMiddleware\NegotiationMiddleware->handle()
StackedHttpKernel.php:51, Drupal\Core\StackMiddleware\StackedHttpKernel->handle()
DrupalKernel.php:686, Drupal\Core\DrupalKernel->handle()
index.php:19, {main}()
```CustomAccessCheck.php:65, Drupal\Core\Access\CustomAccessCheck->access()
AccessManager.php:160, call_user_func_array:{/var/www/html/web/core/lib/Drupal/Core/Access/AccessManager.php:160}()
AccessManager.php:160, Drupal\Core\Access\AccessManager->performCheck()
AccessManager.php:136, Drupal\Core\Access\AccessManager->check()
AccessManager.php:113, Drupal\Core\Access\AccessManager->checkRequest()
AccessAwareRouter.php:104, Drupal\Core\Routing\AccessAwareRouter->checkAccess()
AccessAwareRouter.php:89, Drupal\Core\Routing\AccessAwareRouter->matchRequest()
PathBasedBreadcrumbBuilder.php:225, Drupal\system\PathBasedBreadcrumbBuilder->getRequestForPath()
PathBasedBreadcrumbBuilder.php:171, Drupal\system\PathBasedBreadcrumbBuilder->build()
PathBasedBreadcrumbBuilder.php:21, Drupal\bwp_base\Breadcrumb\PathBasedBreadcrumbBuilder->build()
BreadcrumbManager.php:83, Drupal\Core\Breadcrumb\BreadcrumbManager->build()
SystemBreadcrumbBlock.php:72, Drupal\system\Plugin\Block\SystemBreadcrumbBlock->build()
BlockPluginViewBuilder.php:218, Drupal\block_plugin_view_builder\BlockPluginViewBuilder::preRender()
DoTrustedCallbackTrait.php:101, call_user_func_array:{/var/www/html/web/core/lib/Drupal/Core/Security/DoTrustedCallbackTrait.php:101}()
DoTrustedCallbackTrait.php:101, Drupal\Core\Render\Renderer->doTrustedCallback()
Renderer.php:788, Drupal\Core\Render\Renderer->doCallback()
Renderer.php:374, Drupal\Core\Render\Renderer->doRender()
Renderer.php:446, Drupal\Core\Render\Renderer->doRender()
Renderer.php:204, Drupal\Core\Render\Renderer->render()
TwigExtension.php:477, Drupal\Core\Template\TwigExtension->escapeFilter()
15:82, __TwigTemplate_d8721f0f6aebbd5bb04850254f3787b8->block_block_content()
Template.php:171, Twig\Template->displayBlock()
15:61, __TwigTemplate_35b0ad2b37afc89a549c145babd54d0a->doDisplay()
Template.php:394, Twig\Template->displayWithErrorHandling()
Template.php:367, Twig\Template->display()
15:45, __TwigTemplate_d8721f0f6aebbd5bb04850254f3787b8->doDisplay()
Template.php:394, Twig\Template->displayWithErrorHandling()
Template.php:367, Twig\Template->display()
Template.php:379, Twig\Template->render()
TemplateWrapper.php:40, Twig\TemplateWrapper->render()
twig.engine:53, twig_render_template()
ThemeManager.php:372, Drupal\Core\Theme\ThemeManager->render()
Renderer.php:433, Drupal\Core\Render\Renderer->doRender()
Renderer.php:446, Drupal\Core\Render\Renderer->doRender()
Renderer.php:204, Drupal\Core\Render\Renderer->render()
TwigExtension.php:477, Drupal\Core\Template\TwigExtension->escapeFilter()
15:92, __TwigTemplate_0393aa411e8c560ed6e7ffe204e1d3df->doDisplay()
Template.php:394, Twig\Template->displayWithErrorHandling()
Template.php:367, Twig\Template->display()
Template.php:379, Twig\Template->render()
TemplateWrapper.php:40, Twig\TemplateWrapper->render()
twig.engine:53, twig_render_template()
ThemeManager.php:372, Drupal\Core\Theme\ThemeManager->render()
Renderer.php:433, Drupal\Core\Render\Renderer->doRender()
Renderer.php:204, Drupal\Core\Render\Renderer->render()
TwigExtension.php:477, Drupal\Core\Template\TwigExtension->escapeFilter()
15:86, __TwigTemplate_6a92af3dbed8ce6ee06c2517506d319e->doDisplay()
Template.php:394, Twig\Template->displayWithErrorHandling()
Template.php:367, Twig\Template->display()
Template.php:379, Twig\Template->render()
TemplateWrapper.php:40, Twig\TemplateWrapper->render()
twig.engine:53, twig_render_template()
ThemeManager.php:372, Drupal\Core\Theme\ThemeManager->render()
Renderer.php:433, Drupal\Core\Render\Renderer->doRender()
Renderer.php:204, Drupal\Core\Render\Renderer->render()
HtmlRenderer.php:158, Drupal\Core\Render\MainContent\HtmlRenderer->Drupal\Core\Render\MainContent\{closure:/var/www/html/web/core/lib/Drupal/Core/Render/MainContent/HtmlRenderer.php:153-159}()
Renderer.php:580, Drupal\Core\Render\Renderer->executeInRenderContext()
HtmlRenderer.php:159, Drupal\Core\Render\MainContent\HtmlRenderer->renderResponse()
MainContentViewSubscriber.php:90, Drupal\Core\EventSubscriber\MainContentViewSubscriber->onViewRenderArray()
ContainerAwareEventDispatcher.php:111, call_user_func:{/var/www/html/web/core/lib/Drupal/Component/EventDispatcher/ContainerAwareEventDispatcher.php:111}()
ContainerAwareEventDispatcher.php:111, Drupal\Component\EventDispatcher\ContainerAwareEventDispatcher->dispatch()
HttpKernel.php:168, Symfony\Component\HttpKernel\HttpKernel->handleRaw()
HttpKernel.php:74, Symfony\Component\HttpKernel\HttpKernel->handle()
Session.php:58, Drupal\Core\StackMiddleware\Session->handle()
KernelPreHandle.php:48, Drupal\Core\StackMiddleware\KernelPreHandle->handle()
PageCache.php:106, Drupal\page_cache\StackMiddleware\PageCache->pass()
PageCache.php:85, Drupal\page_cache\StackMiddleware\PageCache->handle()
Cors.php:53, Asm89\Stack\Cors->handle()
ReverseProxyMiddleware.php:48, Drupal\Core\StackMiddleware\ReverseProxyMiddleware->handle()
NegotiationMiddleware.php:51, Drupal\Core\StackMiddleware\NegotiationMiddleware->handle()
StackedHttpKernel.php:51, Drupal\Core\StackMiddleware\StackedHttpKernel->handle()
DefaultExceptionHtmlSubscriber.php:166, Drupal\Core\EventSubscriber\DefaultExceptionHtmlSubscriber->makeSubrequest()
DefaultExceptionHtmlSubscriber.php:122, Drupal\Core\EventSubscriber\DefaultExceptionHtmlSubscriber->on404()
HttpExceptionSubscriberBase.php:109, Drupal\Core\EventSubscriber\HttpExceptionSubscriberBase->onException()
ContainerAwareEventDispatcher.php:111, call_user_func:{/var/www/html/web/core/lib/Drupal/Component/EventDispatcher/ContainerAwareEventDispatcher.php:111}()
ContainerAwareEventDispatcher.php:111, Drupal\Component\EventDispatcher\ContainerAwareEventDispatcher->dispatch()
HttpKernel.php:221, Symfony\Component\HttpKernel\HttpKernel->handleThrowable()
HttpKernel.php:89, Symfony\Component\HttpKernel\HttpKernel->handle()
Session.php:58, Drupal\Core\StackMiddleware\Session->handle()
KernelPreHandle.php:48, Drupal\Core\StackMiddleware\KernelPreHandle->handle()
PageCache.php:106, Drupal\page_cache\StackMiddleware\PageCache->pass()
PageCache.php:85, Drupal\page_cache\StackMiddleware\PageCache->handle()
Cors.php:53, Asm89\Stack\Cors->handle()
ReverseProxyMiddleware.php:48, Drupal\Core\StackMiddleware\ReverseProxyMiddleware->handle()
NegotiationMiddleware.php:51, Drupal\Core\StackMiddleware\NegotiationMiddleware->handle()
StackedHttpKernel.php:51, Drupal\Core\StackMiddleware\StackedHttpKernel->handle()
DefaultExceptionHtmlSubscriber.php:166, Drupal\Core\EventSubscriber\DefaultExceptionHtmlSubscriber->makeSubrequest()
DefaultExceptionHtmlSubscriber.php:122, Drupal\Core\EventSubscriber\DefaultExceptionHtmlSubscriber->on404()
HttpExceptionSubscriberBase.php:109, Drupal\Core\EventSubscriber\HttpExceptionSubscriberBase->onException()
ContainerAwareEventDispatcher.php:111, call_user_func:{/var/www/html/web/core/lib/Drupal/Component/EventDispatcher/ContainerAwareEventDispatcher.php:111}()
ContainerAwareEventDispatcher.php:111, Drupal\Component\EventDispatcher\ContainerAwareEventDispatcher->dispatch()
HttpKernel.php:221, Symfony\Component\HttpKernel\HttpKernel->handleThrowable()
HttpKernel.php:89, Symfony\Component\HttpKernel\HttpKernel->handle()
Session.php:58, Drupal\Core\StackMiddleware\Session->handle()
KernelPreHandle.php:48, Drupal\Core\StackMiddleware\KernelPreHandle->handle()
PageCache.php:106, Drupal\page_cache\StackMiddleware\PageCache->pass()
PageCache.php:85, Drupal\page_cache\StackMiddleware\PageCache->handle()
Cors.php:53, Asm89\Stack\Cors->handle()
ReverseProxyMiddleware.php:48, Drupal\Core\StackMiddleware\ReverseProxyMiddleware->handle()
NegotiationMiddleware.php:51, Drupal\Core\StackMiddleware\NegotiationMiddleware->handle()
StackedHttpKernel.php:51, Drupal\Core\StackMiddleware\StackedHttpKernel->handle()
DrupalKernel.php:686, Drupal\Core\DrupalKernel->handle()
index.php:19, {main}()

**Comment by Hydraner** [comment]
```
CustomAccessCheck.php:65, Drupal\Core\Access\CustomAccessCheck->access()
AccessManager.php:160, call_user_func_array:{/var/www/html/web/core/lib/Drupal/Core/Access/AccessManager.php:160}()
AccessManager.php:160, Drupal\Core\Access\AccessManager->performCheck()
AccessManager.php:136, Drupal\Core\Access\AccessManager->check()
AccessManager.php:113, Drupal\Core\Access\AccessManager->checkRequest()
AccessAwareRouter.php:104, Drupal\Core\Routing\AccessAwareRouter->checkAccess()
AccessAwareRouter.php:89, Drupal\Core\Routing\AccessAwareRouter->matchRequest()
PathBasedBreadcrumbBuilder.php:225, Drupal\system\PathBasedBreadcrumbBuilder->getRequestForPath()
PathBasedBreadcrumbBuilder.php:171, Drupal\system\PathBasedBreadcrumbBuilder->build()
PathBasedBreadcrumbBuilder.php:21, Drupal\bwp_base\Breadcrumb\PathBasedBreadcrumbBuilder->build()
BreadcrumbManager.php:83, Drupal\Core\Breadcrumb\BreadcrumbManager->build()
SystemBreadcrumbBlock.php:72, Drupal\system\Plugin\Block\SystemBreadcrumbBlock->build()
BlockPluginViewBuilder.php:218, Drupal\block_plugin_view_builder\BlockPluginViewBuilder::preRender()
DoTrustedCallbackTrait.php:101, call_user_func_array:{/var/www/html/web/core/lib/Drupal/Core/Security/DoTrustedCallbackTrait.php:101}()
DoTrustedCallbackTrait.php:101, Drupal\Core\Render\Renderer->doTrustedCallback()
Renderer.php:788, Drupal\Core\Render\Renderer->doCallback()
Renderer.php:374, Drupal\Core\Render\Renderer->doRender()
Renderer.php:446, Drupal\Core\Render\Renderer->doRender()
Renderer.php:204, Drupal\Core\Render\Renderer->render()
TwigExtension.php:477, Drupal\Core\Template\TwigExtension->escapeFilter()
15:82, __TwigTemplate_d8721f0f6aebbd5bb04850254f3787b8->block_block_content()
Template.php:171, Twig\Template->displayBlock()
15:61, __TwigTemplate_35b0ad2b37afc89a549c145babd54d0a->doDisplay()
Template.php:394, Twig\Template->displayWithErrorHandling()
Template.php:367, Twig\Template->display()
15:45, __TwigTemplate_d8721f0f6aebbd5bb04850254f3787b8->doDisplay()
Template.php:394, Twig\Template->displayWithErrorHandling()
Template.php:367, Twig\Template->display()
Template.php:379, Twig\Template->render()
TemplateWrapper.php:40, Twig\TemplateWrapper->render()
twig.engine:53, twig_render_template()
ThemeManager.php:372, Drupal\Core\Theme\ThemeManager->render()
Renderer.php:433, Drupal\Core\Render\Renderer->doRender()
Renderer.php:446, Drupal\Core\Render\Renderer->doRender()
Renderer.php:204, Drupal\Core\Render\Renderer->render()
TwigExtension.php:477, Drupal\Core\Template\TwigExtension->escapeFilter()
15:92, __TwigTemplate_0393aa411e8c560ed6e7ffe204e1d3df->doDisplay()
Template.php:394, Twig\Template->displayWithErrorHandling()
Template.php:367, Twig\Template->display()
Template.php:379, Twig\Template->render()
TemplateWrapper.php:40, Twig\TemplateWrapper->render()
twig.engine:53, twig_render_template()
ThemeManager.php:372, Drupal\Core\Theme\ThemeManager->render()
Renderer.php:433, Drupal\Core\Render\Renderer->doRender()
Renderer.php:204, Drupal\Core\Render\Renderer->render()
TwigExtension.php:477, Drupal\Core\Template\TwigExtension->escapeFilter()
15:86, __TwigTemplate_6a92af3dbed8ce6ee06c2517506d319e->doDisplay()
Template.php:394, Twig\Template->displayWithErrorHandling()
Template.php:367, Twig\Template->display()
Template.php:379, Twig\Template->render()
TemplateWrapper.php:40, Twig\TemplateWrapper->render()
twig.engine:53, twig_render_template()
ThemeManager.php:372, Drupal\Core\Theme\ThemeManager->render()
Renderer.php:433, Drupal\Core\Render\Renderer->doRender()
Renderer.php:204, Drupal\Core\Render\Renderer->render()
HtmlRenderer.php:158, Drupal\Core\Render\MainContent\HtmlRenderer->Drupal\Core\Render\MainContent\{closure:/var/www/html/web/core/lib/Drupal/Core/Render/MainContent/HtmlRenderer.php:153-159}()
Renderer.php:580, Drupal\Core\Render\Renderer->executeInRenderContext()
HtmlRenderer.php:159, Drupal\Core\Render\MainContent\HtmlRenderer->renderResponse()
MainContentViewSubscriber.php:90, Drupal\Core\EventSubscriber\MainContentViewSubscriber->onViewRenderArray()
ContainerAwareEventDispatcher.php:111, call_user_func:{/var/www/html/web/core/lib/Drupal/Component/EventDispatcher/ContainerAwareEventDispatcher.php:111}()
ContainerAwareEventDispatcher.php:111, Drupal\Component\EventDispatcher\ContainerAwareEventDispatcher->dispatch()
HttpKernel.php:168, Symfony\Component\HttpKernel\HttpKernel->handleRaw()
HttpKernel.php:74, Symfony\Component\HttpKernel\HttpKernel->handle()
Session.php:58, Drupal\Core\StackMiddleware\Session->handle()
KernelPreHandle.php:48, Drupal\Core\StackMiddleware\KernelPreHandle->handle()
PageCache.php:106, Drupal\page_cache\StackMiddleware\PageCache->pass()
PageCache.php:85, Drupal\page_cache\StackMiddleware\PageCache->handle()
Cors.php:53, Asm89\Stack\Cors->handle()
ReverseProxyMiddleware.php:48, Drupal\Core\StackMiddleware\ReverseProxyMiddleware->handle()
NegotiationMiddleware.php:51, Drupal\Core\StackMiddleware\NegotiationMiddleware->handle()
StackedHttpKernel.php:51, Drupal\Core\StackMiddleware\StackedHttpKernel->handle()
DefaultExceptionHtmlSubscriber.php:166, Drupal\Core\EventSubscriber\DefaultExceptionHtmlSubscriber->makeSubrequest()
DefaultExceptionHtmlSubscriber.php:122, Drupal\Core\EventSubscriber\DefaultExceptionHtmlSubscriber->on404()
HttpExceptionSubscriberBase.php:109, Drupal\Core\EventSubscriber\HttpExceptionSubscriberBase->onException()
ContainerAwareEventDispatcher.php:111, call_user_func:{/var/www/html/web/core/lib/Drupal/Component/EventDispatcher/ContainerAwareEventDispatcher.php:111}()
ContainerAwareEventDispatcher.php:111, Drupal\Component\EventDispatcher\ContainerAwareEventDispatcher->dispatch()
HttpKernel.php:221, Symfony\Component\HttpKernel\HttpKernel->handleThrowable()
HttpKernel.php:89, Symfony\Component\HttpKernel\HttpKernel->handle()
Session.php:58, Drupal\Core\StackMiddleware\Session->handle()
KernelPreHandle.php:48, Drupal\Core\StackMiddleware\KernelPreHandle->handle()
PageCache.php:106, Drupal\page_cache\StackMiddleware\PageCache->pass()
PageCache.php:85, Drupal\page_cache\StackMiddleware\PageCache->handle()
Cors.php:53, Asm89\Stack\Cors->handle()
ReverseProxyMiddleware.php:48, Drupal\Core\StackMiddleware\ReverseProxyMiddleware->handle()
NegotiationMiddleware.php:51, Drupal\Core\StackMiddleware\NegotiationMiddleware->handle()
StackedHttpKernel.php:51, Drupal\Core\StackMiddleware\StackedHttpKernel->handle()
DrupalKernel.php:686, Drupal\Core\DrupalKernel->handle()
index.php:19, {main}()
```

**Comment by jensschuppe** [comment]
This will need refactoring of the form constructor, moving all initialization stuff into a separate method, so that access checks will be done by the defined access callback and do not rely on exceptions, that do not seem to be caught reliably.

Current workaround is to not include breadcrumbs in 404 pages.

**Comment by jensschuppe** [comment]
Fixed via #19.
